.env.example

```
@@ -1,6 +1,7 @@
 DATABASE_URL="postgresql://postgres:password@localhost:5432/db-name"
 BETTER_AUTH_SECRET="change-me-generate-with-openssl-rand-base64-32"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)